doc/source/contributing_code.rst

```diff
@@ -625,6 +627,8 @@ A typical function docstring looks like the following:
             """
             return True
 
+Notes: Funtions should consistently follow the type hinting recommendations throughout Wherever needed following `type hinting pep-0484 <https://www.python.org/dev/peps/pep-0484/>`_
+
```

Comment:
thanks @robotjellyzone, can you please fix the punctuation/grammar? I would replace `Wherever needed` with `by`

Reply:
Hi, I have done the changes. Please check thanks for your review:)